### PR TITLE
[Pages] Document service bindings

### PR DESCRIPTION
**Pull request by WalshyDev**
TODO: Add image once dash is finalised

### CODE REVIEW

File: content/pages/platform/functions/_index.md [hunk 1]
```diff
@@ -419,6 +419,15 @@ export async function onRequestGet({ env }) {
 }
 ```
 
+### Service binding
+
+Service bindings allow for you to call a Worker from within your Function with zero latency. You can add a service binding to your project by going to **Account Home** > **Pages** > **your Pages project** > **Settings** > **Functions** > **Service binding**. Select a Service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.
+
+![Editing a Service Binding and adding a Variable name](/pages/platform/media/)
```
Comment (deadlypants1973) [inline content/pages/platform/functions/_index.md]:
```suggestion
![Follow the instructions above to edit a Service Binding and add a Variable name](/pages/platform/media/)
```

File: content/pages/platform/functions/_index.md [hunk 2]
```diff
@@ -419,6 +419,15 @@ export async function onRequestGet({ env }) {
...
+
+### Using Service bindings locally
```
Comment (deadlypants1973) [inline content/pages/platform/functions/_index.md]:
```suggestion
### Use Service bindings locally
```

File: content/pages/platform/functions/_index.md [hunk 3]
```diff
@@ -419,6 +419,15 @@ export async function onRequestGet({ env }) {
 }
 ```
 
+### Service binding
```
Comment (deadlypants1973) [inline content/pages/platform/functions/_index.md:422]:
```suggestion
### Service bindings
```

File: content/pages/platform/functions/_index.md [hunk 4]
```diff
@@ -419,6 +419,15 @@ export async function onRequestGet({ env }) {
...
+
+Service bindings allow for you to call a Worker from within your Function with zero latency. You can add a service binding to your project by going to **Account Home** > **Pages** > **your Pages project** > **Settings** > **Functions** > **Service binding**. Select a Service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/_index.md]:
```suggestion
Service bindings allow for you to call a Worker from within your Function with zero latency. To add a service binding to your project: 

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
2. In **Account Home**, select **Pages** > **your Pages project** > **Settings** > **Functions** > **Service bindings**. 
3. Select a service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/platform/functions/_index.md (hunk 3 above)
Comment (WalshyDev) [reply content/pages/platform/functions/_index.md]:
```suggestion
### Service binding
```

File: content/pages/platform/functions/_index.md [hunk 5]
```diff
@@ -419,6 +419,15 @@ export async function onRequestGet({ env }) {
 }
 ```
 
+### Service bindings
```
Comment (WalshyDev) [inline content/pages/platform/functions/_index.md]:
```suggestion
### Service binding
```

File: content/pages/platform/functions/_index.md (hunk 3 above)
Comment (WalshyDev) [reply content/pages/platform/functions/_index.md:422]: We use a singular for all the others too

File: content/pages/platform/functions/_index.md [hunk 6]
```diff
@@ -419,6 +419,19 @@ export async function onRequestGet({ env }) {
...
 
+### Service binding
+
+Service bindings allow for you to call a Worker from within your Function with zero latency. To add a service binding to your project: 
+
+1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
+2. In **Account Home**, select **Pages** > **your Pages project** > **Settings** > **Functions** > **Service bindings**. 
+3. Select a service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.
+
+![Follow the instructions above to edit a Service Binding and add a Variable name](/pages/platform/media/)
```
Comment (WalshyDev) [inline content/pages/platform/functions/_index.md]:
```suggestion
```

**Pull request merged**